scapy/contrib/bgp.py

```diff
@@ -117,25 +119,50 @@ def i2m(self, pkt, i):
         """"Internal" (IP as bytes, mask as int) to "machine"
         representation."""
         mask, ip = i
-        ip = socket.inet_aton(ip)
+        ip = socket.inet_aton(str(ip))
```

Comment:
why? (add a comment)

Reply:
inet_aton expects a string representation of an ip address.
I'm ensuring that is the case, and any errors will make this obvious.